### ISSUE TITLE
Failure with php 7.2

### ISSUE THREAD

**Issue by hemberger** [theseer/phpdox#322]
Using the `php:7` docker image (which currently uses `PHP_VERSION=7.2.0`), I get the following error when trying to run phpdox:
```
PHP Version: 7.2.0 (Linux)
PHPDox Version: 0.10.1
ErrorException: E_WARNING 
Location: /root/.composer/vendor/theseer/phpdox/src/shared/FileInfoCollection.php (Line 67)

count(): Parameter must be an array or an object that implements Countable

#0 /root/.composer/vendor/theseer/phpdox/src/shared/FileInfoCollection.php(67): count()
#1 /root/.composer/vendor/theseer/phpdox/src/shared/FileInfoCollection.php(53): TheSeer\phpDox\FileInfoCollection->count()
#2 /root/.composer/vendor/theseer/phpdox/src/bootstrap/Bootstrap.php(56): TheSeer\phpDox\FileInfoCollection->valid()
#3 /root/.composer/vendor/theseer/phpdox/src/CLI.php(130): TheSeer\phpDox\Bootstrap->load()
#4 /root/.composer/vendor/theseer/phpdox/phpdox(79): TheSeer\phpDox\CLI->run()
```

Last success I had was with `PHP_VERSION=7.1.12` (the `php:7.1` docker image).

I assume this will be fixed by #317  ?

**Comment by hemberger** [comment]
I think I may have overlooked that this was already fixed by https://github.com/theseer/phpdox/commit/e521bc962984e24d63338071cde1128dd4d421ba. If we are just waiting on the version installed by Composer to be bumped, please close this, and my apologies for the duplication.

**Comment by theseer** [comment]
I indeed should bump the version to release the fix. Thank you for the reminder.

**Comment by dehrk** [comment]
Any idea when this bug will be fixed? Thanks.

**Comment by theseer** [comment]
Fix is part of the 0.11 release.
